docs/statistics.md

```diff
@@ -1,269 +1,425 @@
-packets are ignored.
+The total number of sent ACK packets. Available for sender.
 
-## pktRetransTotal
+TODO: 
 
-The total number of retransmitted packets. Calculated on the sender's side only.
-Not exchanged with the receiver.
+- possible mistake, see pktSentNAKTotal -> receiver side? -> Maxim should check (Available for sender. -> receiver)
 
-## pktSentACKTotal
+### pktRecvACKTotal
 
-The total number of sent ACK packets. Applicable for data sender.
+The total number of received ACK packets. Available for receiver.
 
-## pktRecvACKTotal
+TODO: Possible mistake, see pktSentNAKTotal. -> Maxim to check
 
-The total number of received ACK packets. Applicable for data receiver.
+### pktSentNAKTotal
 
-## pktSentNAKTotal
+The total number of NAK packets received by the sender from the receiver. Essentially means LOSS reports. Available for sender.
 
-The total number of NAK (Not Acknowledged) packets sent. Essentially means LOSS 
-reports. Applicable for data sender.
+TODO: 
 
-## pktRecvNAKTotal
+- abbreviation
+- sent -> received - right? or if it's sent than it's receiver statistics. Correct in the table as well. -> Maxim should check
 
-The total number of NAK (Not Acknowledged) packets received. Essentially means 
-LOSS reports. Applicable for data receiver.
+### pktRecvNAKTotal
 
-## usSndDurationTotal
+The total number of NAK packets sent back to the sender by the receiver. Essentially means LOSS reports. Available for receiver.
+
+TODO: received -> sent? Or if it's received, than it's sender statistics. Correct in the table as well. -> Maxim to check
+
+### usSndDurationTotal
 
 The total accumulated time in microseconds, during which the SRT sender has some 
-data to transmit, including packets that were sent, but is waiting for acknowledgement.
-In other words, the total accumulated duration in microseconds when there was something 
-to deliver (non-empty senders' buffer).
-Applicable for data sender.
+data to transmit, including packets that have been sent, but not yet acknowledged. In other words, the total accumulated duration in microseconds when there was something to deliver (non-empty senders' buffer). Available for sender.
 
-## pktSndDropTotal
+### pktSndDropTotal
 
-The number of "too late to send" packets dropped by sender (refer to `TLPKTDROP`).
+The total number of "too late to send" packets dropped by the sender (refer to `SRTO_TLPKTDROP` in [API.md](API.md)). Available for sender.
+
+TODO: Update what's below -> Maxim to check -> issue on Github - update RFC and here put the link
 
 The total delay before `TLPKTDROP` is triggered consists of the `SRTO_PEERLATENCY`, 
 plus `SRTO_SNDDROPDELAY`, plus 2 * the ACK interval (default ACK interval is 10 ms).
 The delay used is the timespan between the very first packet and the latest packet 
 in the sender's buffer.
 
+### pktRcvDropTotal
+
+TODO: 
 
-## pktRcvDropTotal
+- missing?
+- Revise the text
+- create ticket -> rename back SRTO_TSBPDDELAY -> Maxim to create
```

Comment:
Since introduction of bidirectional live connection, there are two latency settings, one per direction. The `SRTO_TSBPDDELAY` name comes from the times before it. The negotiated (effective) latency in particular direction is available through `SRTO_PEERLATENCY` on the sender side and `SRTO_RCVLATENCY` on the receiver side. The existing `SRTO_LATENCY` option is deprecated and its management is done through some compromise.